packages/components/src/components/post-popover/post-popover.scss

```
@@ -19,10 +22,10 @@
   padding: 0.5em;
 
   min-width: 160px;
-  max-width: 280px;
+  max-width: var(--post-popover-max-width);
```

How about using this rule?

```suggestion
  max-width: min(var(--post-popover-max-width, 280px), 100%);
```

**Breakdown**
- min(`x`, 100%): Ensure that max width does not exceed 100%, no matter what's set in `x`
- var(`x`, 280px): Use `--post-popover-max-width` if it's set, otherwise fall back to 280px as the default max-width

This would enable you to remove the declaration on the host and the max-width on the popovercontainer. It likely also makes the mobile declaration obsolete.

---

It didn't work with the 100% as it was adding a large margin on the right if used in the `min()` but I managed to fix it using 100vw instead